### DISCUSSION TITLE
Commandbar overflow, default selection

### DISCUSSION THREAD

**Discussion by ashutosh-kendurkar** [microsoft/fluentui#26680] [Q&A] [answered]
when the commandbar has overflow actions, if user clicks on the three dots(...) the expanded dropdown menu may have 1 or more actions, as per the [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/) the focus should be on the first item in the dropdown.

existing behavior - in the current commandbar implementation, the focus remains on the three dots

**Comment by micahgodbolt** [comment, marked as answer]
Focus actually does go to the first item, but there is an existing bug in focus rectangles when moving between surfaces. We have an issue logged for it.
